HomeBloodGlucoseMonitor/R4/resources/Questionnaire-R4-HomeBloodGlucoseMonitorFaceToFace.json

```diff
@@ -0,0 +1,506 @@
+              "operator": "=",
+              "answerBoolean": false
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "4",
+      "type": "group",
+      "text": "Patient diagnosis",
+      "item": [
+        {
+          "linkId": "4.1",
+          "text": "Diabetes Mellitus",
+          "type": "open-choice",
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.35",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".DMCodings"
+              }
+            }
+          ],
+          "repeats": "true",
+          "answerOption": []
+        },
+        {
+          "linkId": "4.2",
+          "text": "Other diagnoses",
+          "type": "open-choice",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".OtherDiagnosesCodings"
+              }
+            }
+          ],
+          "repeats": "true",
+          "answerOption": []
+        }
+      ]
+    },
+    {
+      "linkId": "5",
+      "text": "Laboratory validation",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "5.1",
+          "type": "boolean",
+          "text": "Confirming diagnosis of Diabetes Mellitus",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".ConfirmingDiabetesCondition"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "5.2",
+          "type": "display",
+          "text": "Check Home Blood Glucose Lab form",
+          "enableWhen": [
+            {
+              "question": "5.1",
+              "operator": "=",
+              "answerBoolean": true
+            }
+          ]
+        },
+        {
+          "linkId": "5.3",
+          "type": "date",
+          "text": "Diagnosis confirmed at",
+          "enableWhen": [
+            {
+              "question": "5.1",
+              "operator": "=",
+              "answerBoolean": false
+            }
+          ],
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeBloodGlucoseMonitorFaceToFacePrepopulation\".DMConfirmedDate"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
```

Comment:
6,7,8,9 can be replaced by Subject sub-questionnaire. That sub-q does not use the CQL expression AllProcedure. You can add that to sub-q